### ISSUE TITLE
0.12.0 frameless window resize/move error

### ISSUE THREAD

**Issue by bachel** [nwjs/nw.js#2956]
On windows 7 I can't resize or move frameless window.

**Comment by edi9999** [comment]
What's the code you're using ? Maybe a small demo ?

Are you using 	`-webkit-app-region:drag` in your css on the html element ?

**Comment by bachel** [comment]
package.json
```
{
  "name": "TEST",
  "main": "app://test/init.html",
  "window": {
    "toolbar": false,
    "frame": false,
    "show" : true
  }
}
```

init.html
```
<html style="-webkit-app-region:drag">
    <body>
       <div>TEST</div>
    </body>
</html>
```

**Comment by bachel** [comment]
Fixed in 0.12.0 alfa2, thanks :)
